docs/user/troubleshooting.md

```diff
@@ -99,6 +99,230 @@ This is safe to ignore and merely indicates that the etcd bootstrapping is still
 
 The easiest way to get more debugging information from the installer is to check the log file (`.openshift_install.log`) in the install directory. Regardless of the logging level specified, the installer will write its logs in case they need to be inspected retroactively.
 
+### Installer fails to initialize the cluster
+
+The installer uses the [cluster-version-operator] to create all the components of an OpenShift cluster. When the installer fails to initialize the cluster, the most important information is looking at the [clusterversion] and [clusteroperator] objects:
```

Comment:
The Kuberenetes docs seem to prefer PascalCase for these object kinds in text (e.g. "ClusterVersion").  For example, see [the DaemonSet docs][1].  I'd have expected to backtick that as well (`ClusterVersion`, etc.), but the upstream convention seems to be not marking the kinds as code.

[1]: https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/